posthog/queries/session_recordings/session_recording_list.py

```diff
@@ -58,15 +58,16 @@ class SessionRecordingList(EventQuery):
     # First $pageview event in a recording is used to extract metadata (brower, location, etc.) without
     # having to return all events.
     _core_single_pageview_event_query = """
-         SELECT
-            $session_id as pageview_session_id,
-            any(properties) as pageview_properties
+         SELECT DISTINCT ON (session_id)
```

Comment:
NIT: is the `ON` necessary here if it's only a single column?

Reply:
Not sure distinct is necessary at all here as we are grouping by session_id anyways